README.rst

```diff
@@ -37,8 +37,39 @@ This will create a butler repository at ``DATA/``, ingest the raw data into ``HS
 It will also run various checks of the data integrity of the processed output.
 The resulting repository in ``DATA/`` will take up about 18GB.
 
+Debugging ``HSC/runs/ci_hsc``
+-----------------------------
+
+If the run fails on a given task for the primary ``HSC/runs/ci_hsc`` run, it is easy to rerun just that task (after making appropriate fixes or debugging output).
+
+.. code-block:: bash
+
+    pipetask run -b DATA -j 1 -i HSC/runs/ci_hsc -o u/USER/testing -p "${DRP_PIPE_DIR}/pipelines/HSC/DRP-ci_hsc.yaml#taskLabelToRerun" -d "skymap='discrete/ci_hsc' AND tract=0 AND patch=69"
+
+This will pick up everything that had been run prior to the task that failed and rerun the task with the output going to a new collection called ``u/USER/testing``.
+
+After fixing any problems, the processing can be resumed with:
```

Comment:
"...problems, the rest of the processing can be resumed with (note the `..` at the end of the task label):" ?